bcipy/simulator/sim_factory.py

```diff
@@ -3,20 +3,24 @@
 from typing import List
 
 from bcipy.helpers.load import load_json_parameters
-from bcipy.simulator.helpers.data_engine import RawDataEngine, RawDataEngineWrapper
-from bcipy.simulator.helpers.metrics import MetricReferee, RefereeImpl, SimMetrics1Handler
-from bcipy.simulator.helpers.model_handler import SignalModelHandler1, ModelHandler, \
-    SigLmModelHandler1
-from bcipy.simulator.helpers.sampler import Sampler, EEGByLetterSampler
-from bcipy.simulator.helpers.state_manager import StateManager, StateManagerImpl
+from bcipy.simulator.helpers.data_engine import (RawDataEngine,
+                                                 RawDataEngineWrapper)
+from bcipy.simulator.helpers.metrics import (MetricReferee, RefereeImpl,
+                                             SimMetrics1Handler)
+from bcipy.simulator.helpers.model_handler import (ModelHandler,
+                                                   SigLmModelHandler1,
+                                                   SignalModelHandler1)
+from bcipy.simulator.helpers.sampler import EEGByLetterSampler, Sampler
+from bcipy.simulator.helpers.state_manager import (StateManager,
+                                                   StateManagerImpl)
 from bcipy.simulator.sim import SimulatorCopyPhrase
 
 
 class SimulationFactoryV2:
     """ Factory class to create Simulator instances """
 
     @staticmethod
-    def create(data_folder: str, smodel_files: List[str],
+    def create(source_dirs: List[Path], smodel_files: List[str],
```

Comment:
Could this be a different method. I like the option of having a data folder wrapper that the user creates and passes in.